src/applications/auth/containers/AuthApp.jsx

```diff
@@ -46,10 +62,17 @@ export class AuthApp extends React.Component {
   };
 
   renderError = () => {
-    const { code } = this.props.location.query;
+    const { code, auth } = this.props.location.query;
     let alertProps;
 
+    if (auth === 'fail') {
+      recordEvent({
+        event: code ? `login-error-code-${code}` : `login-error-no-code`,
```

Comment:
maybe `login-error-code-none`? so that we could look for `login-error-code-*` in GA?